ibtk/src/lagrangian/FEDataManager.cpp

```diff
@@ -1137,6 +1137,7 @@ FEDataManager::spread(const int f_data_idx,
                 for (unsigned int e_idx = 0; e_idx < num_active_patch_elems; ++e_idx)
                 {
                     Elem* const elem = patch_elems[e_idx];
+                    TBOX_ASSERT(getPatchLevel(elem) == ln);
```

Comment:
Do we want to include a debugging guard? I think that `TBOX_ASSERT` is _not_ a no-op in SAMRAI 2 (but that it is in SAMRAI 3/4).

Reply:
Yeah, lets do this. This check should always pass so in principle it should only be run when we are debugging IBAMR itself.